web/core/components/issues/issue-detail/main-content.tsx

```diff
@@ -155,7 +155,7 @@ export const IssueMainContent: React.FC<Props> = observer((props) => {
             <DescriptionVersionsRoot
               className="flex-shrink-0"
               entityInformation={{
-                createdAt: new Date(issue.created_at),
+                createdAt: new Date(issue.created_at ?? ""),
```

Comment:
This will result in `Invalid Date` error.
```suggestion
                createdAt: issue.created_at ? new Date(issue.created_at) : "",
```

Can we do this instead?